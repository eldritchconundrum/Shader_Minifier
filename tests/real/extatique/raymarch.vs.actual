/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef RAYMARCH_VS_ACTUAL_
# define RAYMARCH_VS_ACTUAL_
# define VAR_DIR "v"

const char *raymarch_vs =
 "varying vec3 v;"
 "void main()"
 "{"
   "vec4 n=ftransform();"
   "gl_Position=n;"
   "v=n.xyz;"
 "}";

#endif // RAYMARCH_VS_ACTUAL_
