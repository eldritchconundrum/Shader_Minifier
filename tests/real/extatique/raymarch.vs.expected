/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef RAYMARCH_VS_EXPECTED_
# define RAYMARCH_VS_EXPECTED_
# define VAR_DIR "v"

const char *raymarch_vs =
 "varying vec3 v;"
 "void main()"
 "{"
   "vec4 n=ftransform();"
   "gl_Position=n;"
   "v=n.xyz;"
 "}";

#endif // RAYMARCH_VS_EXPECTED_
